src/react/account/AccountCreate.jsx

```diff
@@ -1,11 +1,10 @@
 // @flow
-import * as Form from '../ui-elements/FormLayout';
 import { Banner, Button } from '@scality/core-ui';
+import Form, * as F from '../ui-elements/FormLayout';
 import { accountEmailValidation, accountNameValidation, accountQuotaValidation } from '../utils/validator';
 import { clearError, createAccount } from '../actions';
 import { useDispatch, useSelector } from 'react-redux';
 import type { AppState } from '../../types/state';
-import Input from '../ui-elements/Input';
 import React from 'react';
 import { push } from 'connected-react-router';
 import { useInput } from '../utils/hooks';
```

Comment:
This could use more conventional reordering such that external packages comes first (i.e. `import React from 'react'`)

Reply:
I use the `sort-imports` eslint rule (i.e.:  first sorted by the used member syntax and then alphabetically by the first member or alias name) https://eslint.org/docs/rules/sort-imports. Do you know the eslint rule that enforces external packages to come first?